README.md

```diff
@@ -7,7 +7,9 @@ Plain simple build step for setting up your project with [Carthage](https://gith
 You can cache the result of `carthage bootstrap` to make Bitrise faster when building unchanged dependencies. For that you'll have to [set up caching](http://devcenter.bitrise.io/caching/about-caching/) in your Bitrise workflow.
 
 1. add a cache pull step before your Carthage step,
-2. set up a cache push step after your Carthage step and make sure to add `./Carthage -> ./Carthage/Cachefile`.
+2. set up a cache push step after your Carthage step and make sure to add:
+ * If you use Carthage v0.20.0 or later, `./Carthage`.
+ * Otherwise, `./Carthage -> ./Carthage/Cachefile`.
```

Comment:
The version file is this: `./Cartfile.resolved`. It will change only if anything changed in `./Carthage` directory. So you will need to cache `Cartfile.resolved`, and only if it changed `./Carthage` will be cached. If you don't cache `Cartfile.resolved`, it will output a log `No Cartfile.resolved found`, and it will start going through all the dependencies, and only builds will be skipped, and it takes more time usually.